code/game/objects/items/devices/von-krabin.dm

```diff
@@ -56,16 +53,20 @@
 		for(var/stat in stats_buff)
 			hive_minded.stats.removeTempStat(stat, "von-crabbin")
 			hive_minded.stats.changeStat(stat, -30)  // hard to adapt back to normality
-		if(eotp)
-			eotp.power += 10
-			eotp.armaments_rate += 5
-			eotp.max_armaments_points += 5
 	the_hiveminded = null
+	..()
+
+/obj/item/device/von_krabin/nt_sword_handle()
```

Comment:
No ..()?
![4710725](https://user-images.githubusercontent.com/65828539/194721439-2fd9cbd3-1eb5-481f-9e96-528abbd430f5.jpeg)

Reply:
i added the function myself , cope.